docs/bok/Activities/Security-Testing.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Security Testing
+tags: 
+  - OSPO (Role)
+  - Developer (Role)
+  - Security Expert (Role)
+  - Placeholder
```

Comment:
Is this a placeholder entry? Seems pretty well written for that.

Reply:
@psmulovics I think so. We want to allow for additional roles as we define them (security expert could delineate into multiple roles in CISO, vulnerability management, etc)

Reply:
That is creating a tag called 'Placeholder' and adds this document to it :) So it's not a placeholder for the role.

Reply:
Hi yes - I've tagged all the documents "Placeholder" where we need to add some decent content.

For this one, I think there are several roles Jim came up with (CISO, TISO etc) that have might roll under the category of "Security Expert". What do you think?